src/assets/SocialMediaIcons/SocialMediaIcons.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const SocialMediaIconsMap = {
     DISCORD: {
         label: 'Discord',
-        url: 'https://discord.gg/2jepA3',
+        url: 'https://discord.gg/Ay2A8kN',
```

Comment:
How about we make some const out of it, so we can change it only in one place?

Reply:
Sure, I cant seem to find a place to save the constant should I create a module for it, in the constants folder?

Reply:
Yeah you can make a social-media-constants.js file under the constants directory. Might be smart to have all of our links here so we'd only have to edit them in one place